Content.Server/Projectiles/ProjectileSystem.cs

```diff
@@ -33,6 +37,13 @@ private void OnStartCollide(EntityUid uid, ProjectileComponent component, ref St
             return;
 
         var target = args.OtherEntity;
+
+        if (component.CursorTarget != target
+            && TryComp(target, out MobStateComponent? targetState)
+            && (targetState.CurrentState == MobState.Critical || targetState.CurrentState == MobState.Dead)
+            && (!TryComp(target, out BuckleComponent? buckleComponent) || !buckleComponent.Buckled))
+            return;
+
```

Comment:
Shouldn't be hardcoded to mobstate.

Reply:
i've been thinking about this for 2 days now and i have no idea what it means

Reply:
@metalgearsloth